adam/language_specific/english/english_language_generator.py

```diff
@@ -161,9 +169,11 @@ def _translate_object_to_noun(
             lexicon_entry = self._unique_lexicon_entry(
                 _object.ontology_node  # pylint:disable=protected-access
             )
-            # Check if the object is the speaker
+            # Check if the situation object is the speaker
             if IS_SPEAKER in _object.properties:
                 word_form = "I"
+            elif IS_ADDRESSEE in _object.properties:
+                word_form = "You"
```

Comment:
Uses lower-case `You` because we are only using capitalization for invariantly-capitalized things like `Mom` or `I`, for simplicity